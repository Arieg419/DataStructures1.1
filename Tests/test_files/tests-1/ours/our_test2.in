Init 184
PlantTree 36 16
PlantTree 31 6
PlantTree 34 3
PlantTree 2 13
PlantTree 6 23
PlantTree 3 21
PlantTree 30 17
PlantTree 12 12
UpdateRottenFruits 5 2
Quit
