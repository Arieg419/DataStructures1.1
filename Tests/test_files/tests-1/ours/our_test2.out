Init done.
PlantTree: SUCCESS
PlantTree: SUCCESS
PlantTree: SUCCESS
PlantTree: SUCCESS
PlantTree: SUCCESS
PlantTree: SUCCESS
PlantTree: SUCCESS
PlantTree: SUCCESS
0xb072200xb073400xb072b00xb074600xb073d00xb0710000
